### PR TITLE
Feature game server unit tests

### PR DESCRIPTION
**Pull request by bogse**
Add Unit tests folder structure for GameServer.
Removed duplicate SystemTest folder

### CODE REVIEW

File: GameServerUnitTests/CommandsTests/WaltToCommandTests.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using GameServer.Commands;
+
+namespace GameServerUnitTests.CommandsTests
+{
+    [TestClass]
+    public class WaltToCommandTests
```
Comment (ileuca) [inline GameServerUnitTests/CommandsTests/WaltToCommandTests.cs]: typo =>Wal"t"

**Review by ileuca — DISMISSED** [review]
Looks good.

File: GameServerUnitTests/CommandsTests/WaltToCommandTests.cs (hunk 1 above)
Comment (bogse) [reply GameServerUnitTests/CommandsTests/WaltToCommandTests.cs]: Fixed the typo for both class name and .cs file

**Review by ileuca — APPROVED** [review]
Looks good

**Pull request merged**